src/main/java/com/amazon/corretto/hotpatch/Logger.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.corretto.hotpatch;
+
+import java.security.AccessControlException;
+
+public class Logger {
+    public static final String VERBOSE_PROPERTY_NAME = "log4jFixerVerbose";
+    private static boolean verbose = true;
+
+    public static void log(final String message) {
+        if (verbose) {
+            System.out.println(message);
+        }
+    }
+
+    public static void log(final Exception ex) {
+        if (verbose) {
+            ex.printStackTrace(System.out);
+        }
+    }
+
+    public static void setVerbose(String args) {
+        verbose = args == null || args.contains(VERBOSE_PROPERTY_NAME + "=true");
```

Comment:
Verbose is true by default and should be able to be disabled by "log4jFixerVerbose=false" in the args.

Reply:
Fixed this, but not just by flipping. 

Intended behavior of the logging, there are two scenarios here, as a static agent or not.
- As a static agent, we are almost guaranteed to have an empty args. In that scenario, we want to attempt to check the system property. The default value is going to be "verbose on"
- When we are being attached to an already existing vm, we should receive the value from the args. In that scenario, we want to keep that value (we don't want to check the System.property of the target VM, if a system property was set, it was read by the Main class that did the injection and pass it in the args. If we get attached without that value (someone different that `HotPatchMain` is attaching us, then we just go with verbose on.

For the Main class, the decision is simpler, we assume verbose unless we get `-q`, `--quiet` via args or `log4jFixerVerbose=true` in properties.